packages/patternlab/styleguide/source/assets/js/modules/imagePromoLink.js

```diff
@@ -0,0 +1,19 @@
+export default (function (document) {
+    let promos = document.querySelector(".ma__image-promo");
+    Array.prototype.forEachCall(promos, promo => {
+        // Check the unit has a link or not.
+        let titleLink = promo.querySelector(".ma__image-promo__title .js-clickable-link");
+        if (titleLink) {
+            promo.style.cursor = "pointer";
+
+            let down, up, link = titleLink;
+            promo.onmousedown = () => (down = +new Date());
```

Comment:
Can you please explain why we need this script?

Reply:
It's to prevent a redundant click event with more links, but we don't need this anymore.  It's removed.